client/.storybook/theme.js

```diff
@@ -0,0 +1,8 @@
+import { create } from '@storybook/theming';
```

Comment:
This file wasn't generated by the script. I created it as part of the custom theming.

Ref: https://storybook.js.org/docs/react/configure/theming#create-a-theme-quickstart